client/dashboard/app/queries/site.ts

```diff
@@ -1,11 +1,21 @@
+import { notFound } from '@tanstack/react-router';
 import { fetchSite, deleteSite, launchSite, SITE_FIELDS, SITE_OPTIONS } from '../../data/site';
 import { queryClient } from '../query-client';
 import type { Site } from '../../data/site';
 import type { Query } from '@tanstack/react-query';
 
 export const siteBySlugQuery = ( siteSlug: string ) => ( {
 	queryKey: [ 'site-by-slug', siteSlug, SITE_FIELDS, SITE_OPTIONS ],
-	queryFn: () => fetchSite( siteSlug ),
+	queryFn: async () => {
+		try {
+			return await fetchSite( siteSlug );
+		} catch ( e: any ) /* eslint-disable-line @typescript-eslint/no-explicit-any */ {
```

Comment:
We can use a trick below to pass the lint

```ts
try {
  ...
} ( e: { error: string } ) {
  ...
}
```

Reply:
Nice, done, thanks!

Reply:
Ooops cannot.

```
client/dashboard/app/queries/site.ts(12,16): error TS1196: Catch clause variable type annotation must be 'any' or 'unknown' if specified.
```

😂

Reply:
oops 😂